Sources/StreamVideo/WebRTC/v2/SFU/SFUEventAdapter.swift

```diff
@@ -302,40 +294,6 @@ final class SFUEventAdapter: @unchecked Sendable {
         }
     }
 
-    /// Handles a JoinResponse event.
-    ///
-    /// - Parameter event: The JoinResponse event to handle.
-    private func handleJoinResponse(
```

Comment:
We remove the handling from the SFUEventAdapter as the processing of the JoinResponse happens in [WebRTCCoordinator+Joining](https://github.com/GetStream/stream-video-swift/blob/0637c1c8aaaf013c60220a951355a648b53fdb94/Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator%2BJoining.swift#L338). Having it here causes duplicated processing of the event.